src/expr/src/vector_op/timestamptz.rs

```diff
@@ -32,6 +32,14 @@ fn lookup_time_zone(time_zone: &str) -> Result<Tz> {
     })
 }
 
+pub fn timestamptz_view(usecs: i64, time_zone: &str) -> Result<chrono::DateTime<Tz>> {
```

Comment:
Could move to `Timestamptz` as a method after #10477.

Reply:
Yes!